.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 php:
-  - 5.4
+  - 5.5
   - 5.6
   - 7.0
```

Comment:
is there a compelling reason for us to test our code against anything older than php7?